pkg/documents/documents.go

```diff
@@ -240,12 +239,15 @@ func retrieveRawDocument(logger log.Logger, repo DocumentRepository, keeper *sec
 		}
 		defer rdr.Close()
 
-		encryptedDoc, err := ioutil.ReadAll(rdr)
-		if err != nil {
-			logger.Log("documents", "failed reading document from storage bucket", "error", err, "customerID", customerID, "documentID", documentID, "requestID", requestID)
+		encryptedDoc := make([]byte, rdr.Size())
+		if n, err := rdr.Read(encryptedDoc); err != nil || n == 0 {
+			logger.Log("documents", "failed reading document from storage bucket", "error", err, "bytesRead", n, "customerID", customerID, "documentID", documentID, "requestID", requestID)
 			moovhttp.Problem(w, err)
 			return
 		}
+		if err != nil && err != io.EOF {
+
```

Comment:
Do we need something here?

Reply:
Yes. I'm about to push it up. Got ahead of myself.